tendrl/ceph_integration/central_store/__init__.py

```diff
@@ -44,3 +44,6 @@ def save_ecprofile(self, ec_profile):
 
     def save_utilization(self, utilization):
         tendrl_ns.etcd_orm.save(utilization)
+
+    def save_job(self, job):
+        tendrl_ns.etcd_orm.save(job)
```

Comment:
@r0h4n ah i forgot if i add in common then it is not required in all component..